docs/community/welcome-to-the-community.md

```diff
@@ -1,45 +1,44 @@
 ---
 id: welcome-to-the-community
-title: Welcome to the Community
+title: 'Welcome to the Community'
```

Comment:
Perhaps change the title to "Welcome to OpenSauced" or "Welcome to OpenSauced Community" to be more specific?